doc/INSTALL-Win64.txt

```diff
@@ -0,0 +1,178 @@
+	Should work at least on 7
+
+- Microsoft Visual Studio 2013 (Community Edition).
+	Get it from https://www.visualstudio.com/nl-nl/products/visual-studio-community-vs
+	Install in default location with default options, though you can slim it down a lot if wanted.
+
+- CMake.
+	Get it from http://www.cmake.org/download/
+	Install in default location with default options and choose "Add CMake to command path for all users".
+
+- GIT client. 
+	Get one from https://git-for-windows.github.io/
+	Git bash should be enough.
+
+- Qt.
+	Get it from https://www.qt.io/download-open-source/
+	Version 5 should work.
+	Get the online installer and be sure to mark MSVS 2013 64bit version.
+	Install in default location with default options.
+
+- Perl.
+	Get it from http://www.activestate.com/activeperl/downloads
+	Get the 64-bit version, use 5.20.x (newer versions don't have dmake yet in the repo).
+	Install in default location with default options.
+	
+	If you want scripting support, you will also have to do this:
+	Go to C:\Perl64\lib\CORE, make the file config.h not read-only and open it for editing.
+	Find the line that starts with "#define PERL_STATIC_INLINE" and put "/*" in front of it (so it's commented).
+	And of course save the file.
+	
+	Some perl releases miss dmake at install time; if you get a strange error at compilation
+	time, run the following command from cmdline: ppm install dmake
+
+- zlib.
+	Get it from http://zlib.net/
+	Get the source as we need to compile it ourselves (until a 64bit dll is provided).
+	Compiling (replace versions with what you have):
+		Unpack in C:\kvirccomp\; then open a VS x64 Native CMD Prompt and run:
+			cd C:\kvirccomp\zlib-1.2.8
+			"C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" x64
+			nmake -f win32/Makefile.msc AS=ml64 LOC="-DASMV -DASMINF -I." OBJA="inffasx64.obj gvmat64.obj inffas8664.obj"
+
+- OpenSSL.
+	Get it from http://www.openssl.org/source/ to compile yourself
+	or get the non-Light Win64 package from http://slproweb.com/products/Win32OpenSSL.html
+	Compiling (replace versions with what you have):
+		Unpack in C:\kvirccomp\; then open a VS x64 Native CMD Prompt and run:
+			perl Configure VC-WIN64A
+			ms\do_win64a
+			nmake -f ms\ntdll.mak
+			cd out32dll
+			..\ms\test
+		All of the tests should run flawlessly.
+
+		Then run the following as a batch-script in the ${OPENSSL_INCLUDE_DIR}/openssl/:
+			@echo off
+			setlocal enabledelayedexpansion
+			for %%f in (*.h) do (
+			 set /P linkn= < %%f
+			 set linkn=!linkn:/=\!
+			 ren %%f %%f.ori
+			 copy !linkn! %%f
+			)
+
+- Python.
+	Optional for scripting purposes
+	Get it from http://www.activestate.com/activepython/downloads/
+	Get the 64-bit 2.x version as 3.x is not backwards compatible and supported by now.
+	Install in default location with default options.
+
+- Enchant.
+	Optional for spellchecking support
+	Get it from https://ci.appveyor.com/api/buildjobs/kf2efxpaes6uugwg/artifacts/enchant.7z
```

Comment:
That's cheating. Please also point at sources (I had to modify it a bit regarding path where it searches for dictionaries)